app/views/comments/_comment.html.erb

```diff
@@ -43,6 +43,24 @@
                   </span>
                 <% end %>
               <% end %>
+
+              <% if comment.alignment %>
```

Comment:
I think this is related to my previous comment but don't you think it's safer to test if this comment has a parent or not?